package.json

```diff
@@ -1,13 +1,16 @@
 {
   "license": "Apache-2.0",
   "devDependencies": {
-    "@vuepress/client": "2.0.0-rc.0",
-    "@vuepress/plugin-docsearch": "2.0.0-rc.0",
+    "@vuepress/bundler-webpack": "2.0.0-rc.9",
+    "@vuepress/client": "2.0.0-rc.9",
+    "@vuepress/plugin-docsearch": "2.0.0-rc.19",
     "@vuepress/plugin-google-analytics": "2.0.0-rc.15",
-    "@vuepress/utils": "2.0.0-rc.0",
-    "vue": "^3.3.11",
-    "vuepress": "2.0.0-rc.0",
-    "vuepress-plugin-sitemap2": "2.0.0-rc.4"
+    "@vuepress/plugin-sitemap": "2.0.0-rc.3",
+    "@vuepress/theme-default": "2.0.0-rc.20",
+    "@vuepress/utils": "2.0.0-rc.9",
+    "sass-loader": "^14.1.1",
```

Comment:
I don't see any reference to `sass-loader`. Why do we need it?

Reply:
The sass-loader is an optional dependency of @vuepress/bundler-webpack (doesn't get automatically installed) and is needed since there's some .scss files.